components/index/Chart.vue

```diff
@@ -27,6 +27,30 @@
         daysArray.push(thatDay.getDate() + '<br/>' + getMonth(thatDay.getMonth()))
       }
       daysArray.reverse()
+
+      const lastDayFinished = new Date(latest).setHours(23, 59, 59, 999) < Date.now()
+      let data = this.txHistory.map(h => h.value)
+      let finishedDaysLength = days
+      if (!lastDayFinished && days > 0) {
+        const unfinishedDayFraction = (latest - new Date(latest).setHours(0, 0, 0, 0)) / (24 * 60 * 60 * 1000)
+        const lastFinishedDayData = data[data.length - 2]
+        const unfinishedDayData = data.pop()
+        const extrapolated = unfinishedDayData + lastFinishedDayData * (1 - unfinishedDayFraction)
```

Comment:
This simple interpolation doesn't have an issue with huge values at the start of the day